src/mathematics/calculus/latitude-longitude.md

```diff
@@ -0,0 +1,46 @@
+## Distance between two points on the Earth
+
+[![std-badge]][std]
+
+By default, Rust provides mathematical [float methods] such as
+trigonometric functions, square root, conversion functions between
+radians and degrees, and so forth.
+
+The following example will use [`to_radians`], [`sin`], [`cos`],
```

Comment:
There's a lot of passive voice in there.  Would you mind restructuring the sentances to be Subject noun verb rather than indirect object, to-verb noun?  Like THhis.  ['to_radians'] computes the distance in kilometers between two points ...  Your next sentance is fine, because the active voice of defining a struct is hard.  I can help you if active voice is a challenge.

Reply:
Here is a version more with more active voice:

```
The following example computes the distance in kilometers between two
points on the Earth with the [Haversine formula]. Points are expressed
as pairs of latitude and longitude in degrees. Then, [`to_radians`]
converts them in radian. [`sin`], [`cos`], [`powi`] and [`sqrt`]
compute the central angle. Finally, it's possible to calculate the
distance.
```

If you think it can be better, help is welcomed :-) English is not my mother tong.

Reply:
Yeah, that will work.  I appreciate the effort